<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select_mi_power_widget</class>
 <widget class="QWidget" name="select_mi_power_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mi_lb">
       <property name="text">
        <string>微逆编号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="mim_err_edt"/>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="mim_err_lb">
       <property name="text">
        <string>原边故障</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="CDateEdit" name="date_edt" native="true">
       <property name="styleSheet">
        <string notr="true">QDateEdit
{
    border: 1px solid #dbdbdb;
    color: #666666;
    font: 30px &quot;微软雅黑&quot;;
}

QDateEdit:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 40px;

    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QDateEdit::down-arrow {
    image: url(:/icon/icon/calendar.ico);
}

QDateEdit::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mi_edt">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="mis_err_lb">
       <property name="text">
        <string>副边故障</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="mis_err_edt"/>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="date_lb">
       <property name="text">
        <string>日期</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QPushButton" name="select_pb">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="flipover_widget" name="flipover_wd" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QWidget</extends>
   <header location="global">cdateedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>flipover_widget</class>
   <extends>QWidget</extends>
   <header location="global">flipover_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
